oledStuff.h

```diff
@@ -24,37 +24,38 @@ void oled_Print_Msg(uint8_t, String, uint16_t);
 
 static bool buttonState = LOW;
 static uint8_t msgMode = 0;
-
-uint32_t    oledSleepTimer = 120000;
+static bool boolDisplay = true; 
 
 uint8_t     lineHeight, charHeight;
-
+DECLARE_TIMER_MIN(oledSleepTimer, 10);
```

Comment:
.. ik snap/herken mijn eigen code niet meer.
Ik vraag mij ook sterk af of dit nog werkt.